src/grimp/application/usecases.py

```diff
@@ -257,20 +258,15 @@ def _scan_chunks(
     )
 
     number_of_processes = len(chunks)
-    if number_of_processes == 1:
-        # No need to spawn a process if there's only one chunk.
-        [chunk] = chunks
-        return _scan_chunk(import_scanner, exclude_type_checking_imports, chunk)
```

Comment:
No need for this special `number_of_processes == 1` case anymore - joblib will do this automatically. https://joblib.readthedocs.io/en/stable/generated/joblib.Parallel.html#joblib.Parallel

> If 1 is given, no parallel computing code is used at all, and the behavior amounts to a simple python for loop.